starport/cmd/network_request_reject.go

```diff
@@ -0,0 +1,69 @@
+package starportcmd
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/numbers"
+	"github.com/tendermint/starport/starport/services/network"
+)
+
+// NewNetworkRequestReject creates a new request reject
+// command to reject requests for a chain.
+func NewNetworkRequestReject() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "reject [launch-id] [number<,...>]",
+		Aliases: []string{"accept"},
+		Short:   "Reject requests",
+		RunE:    networkRequestRejectHandler,
+		Args:    cobra.ExactArgs(2),
+	}
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetHome())
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	return c
+}
+
+func networkRequestRejectHandler(cmd *cobra.Command, args []string) error {
+	// initialize network common methods
+	nb, err := newNetworkBuilder(cmd)
+	if err != nil {
+		return err
+	}
+	defer nb.Cleanup()
+
+	// parse launch ID
+	launchID, err := strconv.ParseUint(args[0], 10, 64)
+	if err != nil {
+		return errors.Wrap(err, "error parsing launchID")
+	}
+	if launchID == 0 {
+		return errors.New("launch ID must be greater than 0")
```

Comment:
Can we move this check to the `network` package to a func like checkLaunchID() and call it inside Network.X methods that receive a launchID? That way we can reuse some logic.